src/backend/utils/adt/agtype.c

```diff
@@ -7817,6 +7817,95 @@ Datum age_sqrt(PG_FUNCTION_ARGS)
     PG_RETURN_POINTER(agtype_value_to_agtype(&agtv_result));
 }
 
+PG_FUNCTION_INFO_V1(age_cbrt);
+
+Datum age_cbrt(PG_FUNCTION_ARGS)
+{
+
+    // check if input is agtype
+    if (PG_ARGISNULL(0))
```

Comment:
By adding the line in age--1.1.0.sql this is unecesssary

Reply:
Done